app/(kiosk display)/kiosks/[slug]/DepartureItemTime.tsx

```diff
@@ -0,0 +1,36 @@
+import clsx from 'clsx';
+import styles from './DepartureItem.module.css';
+import { DepartureTime } from '../../../../types/kioskDisplayTypes/GroupedRoute';
+import RealTimeIcon from './RealTimeIcon';
+import { useRecoilValue } from 'recoil';
+import { darkModeState } from '../../../../state/kioskState';
+
+interface DepartureItemTimeProps {
+	time: DepartureTime;
```

Comment:
From a clean code perspective, I would recommend changing the name of this so you don't have `time.time`. You can then destruct the time values on line 13.

```suggestion
	departureTime: DepartureTime;
```